packages/aws-cdk-lib/aws-route53/README.md

```diff
@@ -192,11 +192,39 @@ const parentZone = new route53.PublicHostedZone(this, 'HostedZone', {
   zoneName: 'someexample.com',
 });
 const crossAccountRole = new iam.Role(this, 'CrossAccountRole', {
-  // The role name must be predictable
-  roleName: 'MyDelegationRole',
-  // The other account
-  assumedBy: new iam.AccountPrincipal('12345678901'),
-});
+      // The role name must be predictable
+      roleName: 'MyDelegationRole',
+      // The other account
+      assumedBy: new iam.AccountPrincipal('12345678901'),
+      // You can scope down this role policy to be least privileged.
+      // If you want the other account to be able to manage specific records,
+      // you can scope down by resource and/or normalized record names
+      inlinePolicies: {
+        "crossAccountPolicy": new iam.PolicyDocument({
+          statements: [
+            new iam.PolicyStatement({
+              sid: "ListHostedZonesByName",
+              effect: iam.Effect.ALLOW,
+              actions: ["route53:ListHostedZonesByName"],
+              resources: ["*"]
```

Comment:
nit: i want a comma here and in all other places that should have punctuation

Reply:
Ack, should be resolved.